validator/validator-main.protoascii

```diff
@@ -3549,65 +3601,30 @@ tags: {
 # 4.10.5 The input element
 tags: {
```

Comment:
you'll need to duplicate this tagspec, one for AMP and another for other formats that keep `password` in the blacklist.

Reply:
Is it enough to leave this one matching for all formats, and then use the new tagspec below to enable `password` just for `html_format: AMP`?

Reply:
Yes that should be sufficient. Probably good to add a test for it.

Reply:
where should I add the test? I added a test to thevalidator/testdata/feature_tests/forms.html, was there another place to test other formats?

Reply:
amp4ads is here: validator/testdata/amp4ads_feature_tests/...
@choumx should we create a similar directory for amp4email feature tests or are those being created elsewhere?

Reply:
I think a similar directory would be great. No need to add a bunch of tests -- just one that makes sure `input[type="password"]` fails for now.

@raywainman @zhangsu Are there any internal tests for the `amp4email` validator spec?